index.html

```diff
@@ -17,122 +17,154 @@
+      <section id="intro">
+        <div id="content">
+          <div id="text1">
+            <h1>I’m Eddy <br />Glad to see you!</h1>
+          </div>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me
+          </p>
+          <h4>LET’S CONNECT</h4>
+          <ul>
+            <li>
+              <a href="https://twitter.com/eddy56388406">
+                <img src="images/Vector.png" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/">
+                <img src="images/Linkedin icon.png" />
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/EddyShimwa">
+                <img src="images/Vector (1).png" />
+              </a>
+            </li>
+            <li>
+              <a href="N/A"> <img src="images/Vector (2).png" /> </a>
+            </li>
+            <li>
+              <a href="N/A"> <img src="images/Vector (3).png" /> </a>
+            </li>
+          </ul>
         </div>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me
-        </p>
-        <h4>LET’S CONNECT</h4>
-        <ul>
-          <li>
-            <a href="https://twitter.com/eddy56388406">
-              <img src="images/Vector.png" />
-            </a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/">
-              <img src="images/Linkedin icon.png" />
-            </a>
-          </li>
-          <li>
-            <a href="https://github.com/EddyShimwa">
-              <img src="images/Vector (1).png" />
-            </a>
-          </li>
-          <li>
-            <a href="N/A"> <img src="images/Vector (2).png" /> </a>
-          </li>
-          <li>
-            <a href="N/A"> <img src="images/Vector (3).png" /> </a>
-          </li>
-        </ul>
       </section>
       <section id="works">
         <div id="work-item-one">
-            <img src="images/Snapshoot-Portfolio.png">
+          <img id="image" src="images/Snapshoot-Portfolio.png" />
+          <div>
             <h1>Tonic</h1>
-             <ul id="first-list">
-                <li id="canopy">CANOPY</li>
-                <li><img src="images/Counter.png">Back End Dev</li>
-                <li><img src="images/Counter.png"> 2015 </li></ul>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul id="first-list">
+              <li id="canopy">CANOPY</li>
```

Comment:
- [OPTIONAL] Please fix the style to create a responsive page following the provided template. It's important to stick to Figma template. As not required by the previous reviewer I consider it optional.

Reply:
Could you clarify that for me?